src/ontology/mondo-ingest.Makefile

```diff
@@ -163,6 +163,18 @@ $(COMPONENTSDIR)/icd10who.owl: $(TMPDIR)/icd10who_relevant_signature.txt | compo
 		remove -T config/properties.txt --select complement --select properties --trim true \
 		annotate --ontology-iri $(URIBASE)/mondo/sources/icd10who.owl --version-iri $(URIBASE)/mondo/sources/$(TODAY)/icd10who.owl -o $@; fi
 
+# TODO: complete
+$(COMPONENTSDIR)/icd11foundation.owl: $(TMPDIR)/icd11foundation_relevant_signature.txt | component-download-icd11foundation.owl
```

Comment:
## `skos:broaderTransitive` and `skos:narrowerTransitive`
- [ ] @matentzn advise
<details><summary>Details</summary>
<p>

**Context from [property discussion thread](https://github.com/monarch-initiative/mondo-ingest/pull/434#discussion_r1483728680)**
Joe:
> The skos:broaderTransitive and skos:narrowerTransitive don't have any usages.

Nico:
> Well then, ditch.

**Question**
What do you mean by ditch?
It sounds like you want me to remove something, but there are no usages to remove. Do you want me to remove the declarations? But I don't know if that's worth the time / trouble.
Maybe by "ditch" you meant to just not worry about these?

</p>
</details>

Reply:
robot `remove --term skos:broaderTransitive --term skos:narrowerTransitive`